liquidprompt

```diff
@@ -2432,6 +2456,12 @@ _lp_default_theme_prompt() {
         LP_VCS=
     fi
 
+    if _lp_dirstack; then
+        LP_STCK=" $lp_dirstack"
```

Comment:
We can't change the past variable names, but we should make the new ones easier to read. How about
```suggestion
        LP_DIRSTACK=" $lp_dirstack_color"
```